package.json

```diff
@@ -18,11 +18,16 @@
     "test": "mocha test/**/*.test.ts"
   },
   "dependencies": {
+    "@hapi/hapi": "^19.1.0",
+    "@hapi/joi": "^17.1.0",
+    "@types/joi": "^14.3.4",
     "body-parser": "^1.18.2",
     "cookie-parser": "^1.4.4",
     "dotenv": "^8.2.0",
     "express": "^4.15.5",
+    "express-joi-validation": "^4.0.3",
```

Comment:
Seems like something never user? I suggest you remove it.

Reply:
Removed this